help_to_heat/frontdoor/schemas.py

```diff
@@ -86,6 +92,9 @@
 household_pages = {
     "country": ("country",),
     "supplier": ("supplier",),
+    "park-home": ("park_home",),
+    "park-home-main-residence": ("park_home_main_residence",),
+    "park-home-application-closed": ("park-home-application-closed",),
     "bulb-warning-page": ("bulb-warning-page",),
     "utility-warehouse-warning-page": ("utility-warehouse-warning-page",),
     "applications-closed": ("applications-closed",),
```

Comment:
Why the inconsistency between dashes and underscores here?

Reply:
Ahh they should all have underscores 😅